sdk/python/test/sdk_unittest/test_training_client.py

```diff
@@ -0,0 +1,193 @@
+import multiprocessing
```

Comment:
Let's try to keep `test_training_client.py` file in the same location as `training_client.py` as we discussed here: https://github.com/kubeflow/katib/issues/2184#issuecomment-1660475384
E.g. Kubeflow Pipelines SDK already follow this: https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp/client.

Reply:
I have changed the location of the file